pymc3/tests/test_distributions.py

```diff
@@ -218,7 +220,12 @@ def build_model(distfam, valuedomain, vardomains, extra_args=None):
         distfam("value", shape=valuedomain.shape, transform=None, **vals)
     return m
 
-
+def laplace_asymmetric_logpdf(value, symmetry,scale = None):
```

Comment:
Is this taken from scipy? If so I would mention that in the doc-string and also add a TODO that it should be removed once scipy adds it.

Reply:
> Is this taken from scipy? 
yeah @twiecki  , It has been taken from SciPy.

Reply:
OK cool, then just add those notes.

Reply:
Also not properly black formatted.

Reply:
hey @twiecki ,
Wish you a very very **Happy New Year**. 
please do review the PR and  let me know if anything is yet to be added or removed.